plugins/indexer/nexus-indexer-lucene-plugin/src/main/java/org/sonatype/nexus/index/events/IndexerManagerEventInspector.java

```diff
@@ -82,7 +81,7 @@ public void on(final RepositoryItemEventDelete evt) {
     }
   }
 
-  private void inspectForIndexerManager(final Event<?> evt) {
+  private void inspectForIndexerManager(final Object evt) {
```

Comment:
Would prefer `RepositoryItemEvent evt` as the signature here, as that's what the following line assumes.